notebooks/problem-sets/quantum_kittens_notepad.ipynb

```diff
@@ -0,0 +1,349 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantum Kittens Notepad"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "Welcome to the Quantum Kittens notepad. \n",
+    "\n",
+    "This is a coding notepad supplement to the book [Quantum Kittens](http://quantumkittens.online), a non-technical book that teaches quantum computing through stories about cats.\n",
+    "\n",
+    "Also available in [Japanese](http://quantum-kittens-ja.github.io/) and [Spanish](http://quantum-kittens-es.github.io/).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src='https://github.com/quantum-kittens/quantum-kittens.github.io/blob/main/assets/imgs/cover_beta.png?raw=true'/>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Quantum Kittens Credits:\n",
+    "- Author: Radha Pyari Sandhir\n",
+    "- Illustrator: Rashi Chandra\n",
+    "- Japanese Translator: Yuri Kobayashi\n",
+    "- Spanish Translator: Claudia Zendejas-Morales \n",
+    "- The Cats: [Arya, Blade, Luna](https://www.instagram.com/catomictheory/)\n",
+    "- Editor: Suzanne Musin\n",
+    "- Technical Reviewer: James Weaver\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Chapter 2 Qiskit Code\n",
+    "\n",
+    "Simulate a Whiskerton marble using this code! \n",
+    "\n",
+    "This code walks you through creating and running a quantum circuit with a single qubit.\n",
+    "\n",
+    "This code is associated with Quantum Kittens Chapter 2:\n",
+    "- [Chapter 2 - Story - Schrödinger Day](https://quantum-kittens.github.io/posts/CHAPTER-2-Story-Schr%C3%B6dinger-Day/)\n",
+    "- [Chapter 2 Part 2 - Commentary - Qubits, Superposition, and Measurement](https://quantum-kittens.github.io/posts/CHAPTER-2-Part-2-Qubits-Superposition-and-Measurements/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src='https://github.com/quantum-kittens/quantum-kittens.github.io/blob/main/assets/imgs/Marble_Animation.png?raw=true'/>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Importing necessary Qiskit libraries; run this cell first\n",
+    "from qiskit import QuantumCircuit, transpile\n",
+    "from qiskit import BasicAer\n",
+    "from qiskit.visualization import *"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "goals": [
+     {
+      "id": "qiskit-beginner-1"
+     }
+    ],
+    "grader_answer": "circuit",
```

Comment:
this should be `marble_circuit` since that is the variable name used in the code cell

Reply:
@vabarbosa Thanks for spotting this. There's no grader, so I've updated the metadata as follows:

- Changed `grader_id` and `grader_answer` to blank
- Changed id to qk-notepad

Please let me know if this is okay.